srcpkgs/corecollector/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'corecollector'
+pkgname=corecollector
+version=0.3.5
+revision=1
+build_style=meson
+configure_args="-Dcoredump_user=_corecollector -Dcoredump_group=_corecollector"
+conf_files="/etc/corecollector/corecollector.conf"
+hostmakedepends="dmd git pkg-config scdoc"
```

Comment:
Why does it need git during the build? If some submodule, would it be possible to add that to distfiles?

Reply:
https://github.com/Cogitri/corecollector/blob/b179691f603f8a2114d7ab67f5da40ccbb7e0bc3/meson.build#L22

I think it only uses git to determine versioning, and there are no submodules.

Reply:
Does the build fail without git? I didn't test.

Reply:
Yes, but in theory git could be patched out

Reply:
No problem. I think it would be nice to ask upstream to fix, but it's not essential.